oktapam/provider_test.go

```diff
@@ -143,3 +151,54 @@ func getTeamName() string {
 	}
 	return DefaultTestTeam
 }
+
+type compositeDualProviderStruct struct {
+	sdkV2Provider *schema.Provider
+	fwProvider    *fwprovider.OktapamFrameworkProvider
+}
+
+func testAccFrameworkMuxProviders(ctx context.Context, t *testing.T) (context.Context, *compositeDualProviderStruct, map[string]func() (tfprotov6.ProviderServer, error)) {
+	// Init sdkV2 provider
+	sdkV2Provider := Provider()
+	// Init framework provider
+	frameworkProvider := &fwprovider.OktapamFrameworkProvider{}
+
+	// Init mux servers
+	muxServer := testAccFrameworkMuxProvidersServer(ctx, sdkV2Provider, frameworkProvider)
+
+	providers := &compositeDualProviderStruct{
+		sdkV2Provider: sdkV2Provider,
+		fwProvider:    frameworkProvider,
+	}
+	return ctx, providers, muxServer
+}
+
+func testAccFrameworkMuxProvidersServer(ctx context.Context, sdkV2Provider *schema.Provider, fwProvider *fwprovider.OktapamFrameworkProvider) map[string]func() (tfprotov6.ProviderServer, error) {
+	return map[string]func() (tfprotov6.ProviderServer, error){
+		"oktapam": func() (tfprotov6.ProviderServer, error) {
+			// upgrade sdk v2 provider to protocol version 6
+			upgradedSdkProvider, err := tf5to6server.UpgradeServer(
```

Comment:
This code can we moved to oktapam/provider/factory.go as I mentioned in earlier comment.